.github/workflows/centos7-ci.yml

```diff
@@ -0,0 +1,87 @@
+name: CI Centos7
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test_apisix:
+    name: run ci on centos7
+    runs-on: ubuntu-latest
+
+    services:
+      etcd:
+        image: bitnami/etcd:3.4.0
+        ports:
+          - 2379:2379
+          - 2380:2380
+        env:
+          ALLOW_NONE_AUTHENTICATION: yes
+          ETCD_ADVERTISE_CLIENT_URLS: http://0.0.0.0:2379
```

Comment:
where we using this service？

Reply:
https://github.com/apache/apisix/blob/e1a2dcd1d1fbce20441b111c5c1dedc77075b63a/utils/centos7-ci.sh#L62-L65
The test needs to compile apisix dependencies, which need etcd, otherwise prompt error `got malformed version message: "" from etcd`